schema/me/conversation/index.js

```diff
@@ -144,16 +158,29 @@ export const ConversationFields = {
 
   artworks: {
     type: new GraphQLList(ArtworkType),
-    resolve: conversation => {
-      const ids = []
-
-      for (const item of conversation.items) {
+    description: "Only the artworks discussed in the conversation.",
+    resolve: ({ items }) => {
+      const results = []
+      items.forEach(item => {
         if (item.item_type === "Artwork") {
-          ids.push(item.item_id)
+          results.push(item.properties)
```

Comment:
Basically we can just resolve as an `Artwork`, with `properties`. We expose that in the Impulse API, and when we snapshot it at the time of inquiry (and backfilled), it represents the public JSON of the artwork (ie- an already resolved `Artwork`). Easy!